lib/tasks/asset_sync.rake

```diff
@@ -10,21 +9,4 @@ namespace :assets do
       AssetSync.clean
     end
   end
-
-end
-
-if Rake::Task.task_defined?("assets:precompile:nondigest")
```

Comment:
@jbusser I'm not sure I understand the implications of removing this block of code. Any insight based on your previous research?